src/static/css/main.css

```diff
@@ -1,102 +1,126 @@
 .hidden {
```

Comment:
:warning: please remove this from the PR
unrelated changes (formatting)

Reply:
I didn't change the formatting.
I add new CSS elements for the panel where the user choose his option for loading. so. It's related to the PR ^^

Reply:
forget this comment, it applies to code prior force push